modules/images-create-s2i-scripts.adoc

```diff
@@ -127,31 +130,23 @@ EOF
 ----
 
 
-////
-== Using Images with ONBUILD Instructions
-The `ONBUILD` instructions can be found in many official container images. For
-example:
+[id="images-s2i-requirements-using-images-with-onbuild-instructions_{context}"]
+== Using images with ONBUILD instructions
+
+The `ONBUILD` instructions can be found in many official container images. For example:
 
 * Ruby
 * Node.js
-*  Python
+* Python
 
-See the link:https://docs.docker.com/engine/reference/builder/#onbuild[Docker
-documentation] for more information on `ONBUILD`.
-
-Upon startup, S2I detects whether the builder image contains `sh` and `tar` binaries
-which are necessary for the S2I process to inject build inputs.  If the builder image
-does not contain these prerequisites, it will attempt to instead perform a container build
-to layer the inputs.  If the builder image includes `ONBUILD` instructions, S2I
-will instead fail the build because the `ONBUILD` instructions would be executed
-during the layering process, and that equates to performing a generic container
-build which is less secure than an S2I build and requires explicit permissions.
+During startup, S2I detects whether the builder image contains `sh` and `tar` binaries which are necessary for the S2I process to inject build inputs. If the builder image does not contain these prerequisites, it attempts to perform a container build to layer the inputs. If the builder image includes `ONBUILD` instructions, S2I fails the build because the `ONBUILD` instructions are run during the layering process, and that equates to performing a generic container build which is less secure than an S2I build and requires explicit permissions.
```

Comment:
We can drop this warning - buildah can handle `ONBUILD` Instructions just fine, and the initial startup step doesn't happen any more.